### ISSUE TITLE
krypt-pkce is not published

### ISSUE THREAD

**Issue by darronschall** [chRyNaN/krypt#1]
Hello! Thanks for these libraries.

I'm working through an OAuth PKCE implementation and stumbled upon your [SecureRandom](https://github.com/chRyNaN/krypt/blob/develop/krypt-csprng/src/commonMain/kotlin/com.chrynan.krypt.csprng/SecureRandom.kt) wrapper for Kotlin Multiplatform Mobile. Working through the source, I saw there was a [krypt-pkce](https://github.com/chRyNaN/krypt/tree/develop/krypt-pkce) project alongside, but [it does not appear to be published yet](https://repo.repsy.io/mvn/chrynan/public/com/chrynan/krypto/). Any chance you can publish it with the rest of the libraries?

Thanks!

**Comment by darronschall** [comment]
Additionally, for the libraries that _are_ published, the published package name does not match what is in the [`README.md`](../blob/develop/README.md#dependencies).

```
implementation("com.chrynan.krypt:krypt-core:$VERSION")
```

... needs to be ...

```
implementation("com.chrynan.krypto:krypt-core:$VERSION")
```

... in order to use the library.

**Comment by chRyNaN** [comment]
Hi @darronschall Thanks for pointing out this issue. The pkce library wasn't released yet, I got sidetracked and haven't had the opportunity to finish up the base of this library. I'll try and get out another release of the latest tonight or tomorrow. I want to fix the other mentioned issue and update to the latest Kotlin version first. And I'll look into the package naming, most likely a typo.

**Comment by chRyNaN** [comment]
These issues were addressed in version 0.2.0 https://github.com/chRyNaN/krypt/releases/tag/0.2.0

**Comment by darronschall** [comment]
Thank you!
